scalalib/src/mill/scalalib/ScalaModule.scala

```diff
@@ -247,6 +247,20 @@ trait ScalaModule extends mill.Module with TaskModule { outer =>
     )
   }
 
+  def ivyDepsTree() = T.command {
+    import coursier.{Cache, Fetch, Resolution}
+
+    val flattened = ivyDeps().map(depToDependency(_, scalaVersion(), platformSuffix())).toSet
+    val start = Resolution(flattened)
+    val fetch = Fetch.from(repositories, Cache.fetch())
+    val resolution = start.process.run(fetch).unsafePerformSync
+
+    coursier.util.Print.dependencyTree(flattened.toSeq, resolution,
```

Comment:
This generates a string and you do nothing with it?

Reply:
Oh! :open_mouth: I thought it would print it! I'll check it out. Thanks